lib/logger.js

```diff
@@ -177,8 +187,16 @@ Logger.prototype._flush = function(cb) {
     this._req.post(this._url, {
         body: data
         , qs: { now: Date.now() }
-    }, function(err) {
-        if (err) { debug('Encountered an Error in POST Request: %j', err); }
+    }, function(err, httpResponse, body) {
+        if (err) {
+            debug('Encountered an Error in POST Request: %j', err);
+        } else {
+            debug('API success: %j', {
+                lines: sendbuf.ls.map(function (msg) { return msg.line; }),
+                httpStatus: httpResponse.statusCode,
+                body: body
+            });
+        }
```

Comment:
This is possibly the most notable addition -- this gives visibility into when messages are actually sent to the API and whether or not the API actually accepted them.

This can help narrow in on whether problems exist on the sending or receiving end.